styles/_mixins.scss

```diff
@@ -0,0 +1,32 @@
+@mixin section-padding {
```

Comment:
It looks like the specificity of the media queries is overriding the intended vertical padding on the previous apprentices section. I'm also not sure that we want to couple the vertical padding to the breakpoints here. Let's decouple the concerns of vertical and horizontal spacing, and update this mixin so that it is only concerned with horizontal spacing. This involves 3 things:

1. Explicitly set `padding-left` and `padding-right` instead of `padding`
2. Move the `width` and `max-width` declarations into this mixin
3. Rename the mixin to `section-horizontal-spacing`

From there, we should be able to set the previous apprentice padding without it being overridden. Subsequently, if the vertical padding needs to be shared across sections, we can create a `section-vertical-spacing` mixin (if necessary).